test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/DoSection.java

```diff
@@ -306,6 +306,11 @@ void checkWarningHeaders(final List<String> warningHeaders) {
                      * We skip warnings related to types deprecation so that we can continue to run the many
                      * mixed-version tests that used typed APIs.
                      */
+                } else // noinspection StatementWithEmptyBody
```

Comment:
Is this `noinspection` something the build tooling pays attention to, or is this IntelliJ-specific? If the latter, it should be removed.

Reply:
it looks like it is related to intelliJ. I was not aware that we can remove it, will give it a go